src/handlers/sdp/MediaSection.ts

```diff
@@ -248,6 +249,10 @@ export class AnswerMediaSection extends MediaSection
 									codecParameters.usedtx = opusDtx ? 1 : 0;
 								}
 
+								if (!opusNack) {
+									offerCodec!.rtcpFeedback = offerCodec?.rtcpFeedback?.filter(fb => fb.type !== 'nack')
+								}
```

Comment:
AFAIU it should also modify `codecParameters` since those are the ones that will be signaled to mediasoup server.
But I don't really understand what this code is doing. I read:

"If `opusNack` is not set, then remove "nack" from its supported RTCP feedbacks"

Why? Here we are supposed to add features rather than removing them.

Reply:
The `{ type: 'nack' }` is injected into the native RTP capabilities for Chrome74 and Safari12, so we need to remove them if the application doesn't want Opus NACK.

I could invert this code to add NACK instead and not inject NACK into native RTP capabilities, but then it would need to be injected in a few other places that the native RTP capabilities are used. This was the most minimal change I could think of.